locales/templates/app.pot

```diff
@@ -259,6 +255,20 @@ msgid ""
 "sign you out of websites and delete web content that is stored offline."
 msgstr ""
 
+msgctxt "settings_cookies_confirm"
+msgid "Clear now"
```

Comment:
Please provide a localization comment to give context (since this can be translated in different ways depending on what it refers to)

Reply:
Our mistake, we thought these ones had already been localized.  I will fix and reopen.